hasura/user-prefs/migrations/default/1712849109532_elevation_plot_series/down.sql

```diff
@@ -0,0 +1,24 @@
+
+alter table "public"."lucumaUserPreferences" rename column "elevationPlotLunarElevationVisible" to "elevationPlotLunarElevationVisiblee";
```

Comment:
Is the second `e` in `Visiblee` intentional?

Reply:
That was supposed  to be fixed